Plugins/UnityServerPlugin/StreamingUnityServerPlugin.cpp

```diff
@@ -57,6 +62,26 @@ using namespace Toolkit3DLibrary;
 void(__stdcall*s_onInputUpdate)(const char *msg);
 void(__stdcall*s_onLog)(const int level, const char *msg);
 
+static struct FsLogStream : rtc::LogSink
```

Comment:
we should abstract this and consume it across our toolkit behind some config flag, it makes debugging a breeze

Reply:
we should remove this for `release` builds